system_testing/test_bucky_project/services/service_a/pc/scenarios/e2e/pc_e2e.yml

```diff
@@ -80,4 +80,52 @@ cases:
           page: index
           part:
             locate: links
-            num: 1
\ No newline at end of file
+            num: 1
+  - case_name: pc_e2e_6
+    func: pc e2e 6 get element from PageObject
+    desc: pc e2e 6 get element from PageObject
+    procs:
+      - proc: open index
+        exec:
+          operate: go
+          url: http://bucky.net
+      - proc: index
+        exec:
+          operate: click_single_element
+          page: index
+  - case_name: pc_e2e_7
+    func: pc e2e 7 get multiple element from PageObject
+    desc: pc e2e 7 get multiple element from PageObject
+    procs:
+      - proc: open index
+        exec:
+          operate: go
+          url: http://bucky.net
+      - proc: index
+        exec:
+          operate: click_multiple_element
+          page: index
+  - case_name: pc_e2e_8
+    func: pc e2e 8 get element from Verifications
+    desc: pc e2e 8 get element from Verifications
```

Comment:
Maybe 「pc e2e 8 click the single element from Verifications」is better.

Reply:
Fixed!